kafka-quarkus-dev/README.adoc

```diff
@@ -63,7 +63,7 @@ For this quick start, you'll use the Quarkus sample code from the {product} http
 ----
```

Comment:
We aren't renaming this quickstart, can you leave it in the quarkus folder.

Reply:
More context - there will be two quarkus quivkstarts. I have second for SBO